rawdisk/filesystems/detector.py

```diff
@@ -54,8 +54,8 @@ def add_gpt_plugin(self, fs_guid, plugin):
         else:
             self.gpt_plugins[fs_guid] = [plugin, ]
 
-    def detect_mbr(self, filename, offset, fs_id):
-        """Used by rawdisk.reader.Reader to match mbr partitions agains
+    def detect_mbr(self, filename, offset, fs_id, verbose=False):
```

Comment:
I'm working on better logging implementation on new logging branch, it will allow to set logging verbosity level. I will revisit this pr after I complete that work, i think passing verbose argument won't be necessary anymore